src/Storage/Storage/DatalakeGen2/Cmdlet/UpdateAzDataLakeGen2Item.cs

```diff
@@ -46,9 +42,9 @@ public class SetAzDataLakeGen2ItemCommand : StorageCloudBlobCmdletBase
         [ValidateNotNullOrEmpty]
         public string FileSystem { get; set; }
 
-        [Parameter(ValueFromPipeline = true, Position = 1, Mandatory = true, HelpMessage =
+        [Parameter(ValueFromPipeline = true, Position = 1, Mandatory = false, HelpMessage =
```

Comment:
I'm curious how PowerShell handle this positional but optional parameter, we may have a discussion tomorrow.

Reply:
Need update change log.